in_toto/runlib.py

```diff
@@ -47,13 +48,15 @@
   try:
     import subprocess32 as subprocess
   except ImportError:
-    log.warn("POSIX users (Linux, BSD, etc.) are strongly encouraged to"
-        " install and use the much more recent subprocess32")
+    in_toto.settings.COLOR = in_toto.util.detect_colorization(sys.argv)
+    log.warn("POSIX users (Linux, BSD, etc.) are encouraged to"
+        " install the new subprocess32 module"
+        " instead of the version provided in python "
+        + str(sys.version_info[0]) + "." + str(sys.version_info[1]))
```

Comment:
I don't think this change is necessary. As a matter of fact, we'll probably remove this all too prominent warning once we have Python3 support.

Reply:
Alright, the change has been reverted in the latest commit.